addons/mail/static/src/web/chatter.js

```diff
@@ -395,17 +398,18 @@ export class Chatter extends Component {
         this.state.showActivities = !this.state.showActivities;
     }
 
+    async schedule(threadId) {
+        await this.activityService.schedule(this.props.threadModel, threadId);
+        this.load(this.props.threadId, ["activities", "messages"]);
+    }
```

Comment:
Note for master / 17.3: `this.activityService.schedule()` has been replaced by `this.store.scheduleActivity()`

_(fyi, reasoning was that discuss code looked very complex with many services, so we killed most of them)_